### DISCUSSION TITLE
I'm confused with the curation and agreement of document labels

### DISCUSSION THREAD

**Discussion by PotOfJam** [inception-project/inception#4849] [Q&A] [answered]
I have two questions.

1. Why is there no measure available for document label agreement? The only choice is "Choose One" in the drop-down menu.

![INCEpTION - Agreement](https://github.com/inception-project/inception/assets/116809610/ada8c8b7-3ec9-4868-93af-0af1147c364a)

2. Is it possible to see which document label each annotator chose during curation?

![INCEpTION - Curation](https://github.com/inception-project/inception/assets/116809610/920060db-8063-4eb4-a26a-0f16f3465c1b)

I’m not sure if all my annotators are always agreeing on everything or if there's a function I'm not aware of. Additionally, I don't know how to see which label each annotator chose.

**Comment by reckart** [comment, marked as answer]
The agreement measures currently do not support document-level annotations. It still needs to be implemented.

https://github.com/inception-project/inception/pull/4857

**Reply by PotOfJam** [reply]
Okay thanks for the answer, should i create a issue ? or is it something that's already being worked one ? and if it is, is there an ETA ?

**Reply by reckart** [reply]
The PR with the changes to enable it is linked in my comment above. It's not tested much yet, but at a first try it seemed to do something.
